base_layer/core/src/consensus/consensus_encoding/integers.rs

```diff
@@ -47,6 +47,7 @@ macro_rules! consensus_encoding_varint_impl {
     };
 }
 
+consensus_encoding_varint_impl!(u8);
```

Comment:
So this seems wrong for a Byte. Varint is more efficient for integers larger than a byte but for a byte half the time the value is actually 2 bytes.

Reply:
Agreed

Reply:
Yip, changed it